assets/components/contributionSelection/contributionSelection.scss

```diff
@@ -46,6 +46,35 @@
   }
 }
 
+.component-contribution-selection--annual {
+  .component-contribution-selection__type {
+    .component-radio-toggle {
+      border: none;
+    }
+    .component-radio-toggle__button {
+      display: inline-block;
+      border: 0.125rem solid #ffe500;
+      border-radius: 37.5rem;
```

Comment:
Could these be pixel values? They get converted to `rem` as part of the build. This is `2px` and `600px` respectively I think?